docs/modules/ROOT/pages/how-tos/cloudscale/remove_node.adoc

```diff
@@ -18,6 +18,12 @@ Steps to remove a worker node of an OpenShift 4 cluster on https://cloudscale.ch
 * You have admin-level access to the cluster
 * You want to remove an existing worker node in the cluster
 
+== High-level overview
+
+* First, we identify the correct node to remove and drain it.
+* Then, we remove it from Kubernetes.
+* Finally, we remove the associated VMs.
```

Comment:
Though I guess since this is a bullet list you could consider each bullet point a standalone sentence and skip the comma... Huh. Guess it's a special case.
(The comma would be needed if all these sentences came one-after-the-other in the same paragraph)

Reply:
> Though I guess since this is a bullet list you could consider each bullet point a standalone sentence

I guess that's what confused me when reviewing. I don't have hard feelings either way, but I'm still mentally stumbling over the commas in the bullet list.